### PR TITLE
feat: federation stopped system messages [WPB-1815]

### PR DESCRIPTION
**Pull request by Garzas**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

- users removed system message for federation stopped
- federation stopped system messages

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

| Before | After |
| ----------- | ------------ |
|   | federationMemberRemoved  |
|   |  federationStopped |

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [819](https://10.10.124.134/job/AR-build-pipeline/819/) **failed**.

**Comment by AndroidBob** [comment]
Build [820](https://10.10.124.134/job/AR-build-pipeline/820/) **failed**.

File: app/src/main/kotlin/com/wire/android/mapper/MessagePreviewContentMapper.kt [hunk 1]
```diff
@@ -270,6 +270,36 @@ fun MessagePreview.uiLastMessageContent(): UILastMessageContent {
...
+                }
+
+                else -> {
+                    UIText.PluralResource(R.plurals.last_message_other_removed_other_users, otherUsersSize, otherUsersSize)
+                }
+            }
+
+            UILastMessageContent.TextMessage(MessageBody(previewMessageContent))
+        }
+
```
Comment (ohassine) [inline app/src/main/kotlin/com/wire/android/mapper/MessagePreviewContentMapper.kt]: Can you cover this section with test ?

Comment (Garzas) [reply app/src/main/kotlin/com/wire/android/mapper/MessagePreviewContentMapper.kt]: Sure, I didn't start writing any test until kalium PR will be approved: https://github.com/wireapp/kalium/pull/1930

**Comment by AndroidBob** [comment]
Build [828](https://10.10.124.134/job/AR-build-pipeline/828/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.4.0(7200778)(7200778)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.4.0(7200778)(7200778)-staging-compat.apk)

**Review by yamilmedina — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [868](https://10.10.124.134/job/AR-build-pipeline/868/) **failed**.

**Comment by AndroidBob** [comment]
Build [912](https://10.10.124.134/job/AR-build-pipeline/912/) **failed**.

**Comment by AndroidBob** [comment]
Build [923](https://10.10.124.134/job/AR-build-pipeline/923/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.4.0(7218283)(7218283)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.4.0(7218283)(7218283)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [950](https://10.10.124.134/job/AR-build-pipeline/950/) **failed**.

**Review by ohassine — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [952](https://10.10.124.134/job/AR-build-pipeline/952/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.4.0(7252313)(7252313)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.4.0(7252313)(7252313)-staging-compat.apk)

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -179,6 +179,7 @@
     <string name="url_message_details_offline_backends_learn_more" translatable="false">https://support.wire.com/hc/articles/9357718008093-Backend</string>
     <string name="url_self_client_verification_learn_more" translatable="false">https://support.wire.com/hc/en-us/articles/207693005-How-can-I-verify-my-own-devices-</string>
     <string name="url_self_client_fingerprint_learn_more" translatable="false">https://support.wire.com/hc/en-us/articles/207692235-How-can-I-compare-key-fingerprints-</string>
+    <string name="url_federation_support" translatable="false">https://support.wire.com/hc/en-us/categories/4719917054365-Federation</string>
```
Comment (yamilmedina) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="url_federation_support" translatable="false">https://support.wire.com/hc/categories/4719917054365-Federation</string>
```

I think we need to use the `un-localized `version of the link here, we have several actually that we need to remove `/en-us` 😅  (not related to federation)

**Review by yamilmedina — COMMENTED** [review]
I left a comment, about the localization of the link :D

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (Garzas) [reply app/src/main/res/values/strings.xml]: Good catch 👁️

**Comment by AndroidBob** [comment]
Build [965](https://10.10.124.134/job/AR-build-pipeline/965/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.4.0(7253763)(7253763)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.4.0(7253763)(7253763)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [1009](https://10.10.124.134/job/AR-build-pipeline/1009/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.4.0(7269843)(7269843)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.4.0(7269843)(7269843)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [1010](https://10.10.124.134/job/AR-build-pipeline/1010/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.4.0(7270591)(7270591)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.4.0(7270591)(7270591)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [1120](https://10.10.124.134/job/AR-build-pipeline/1120/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.4.0(7319891)(7319891)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.4.0(7319891)(7319891)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [1121](https://10.10.124.134/job/AR-build-pipeline/1121/) **failed**.

**Comment by AndroidBob** [comment]
Build [1124](https://10.10.124.134/job/AR-build-pipeline/1124/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.4.0(7320621)(7320621)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.4.0(7320621)(7320621)-staging-compat.apk)

**Pull request merged**